system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -83,10 +86,18 @@
 
     @include context-menu-button;
 
-    &:hover, &:focus, &.active {
+    &:hover,
+    &:focus,
+    &.active {
```

Comment:
I wonder if we should switch these types of selectors to something like `&:-moz-any(.active, :focus, :hover) {`. The generated css before/after:
```diff
-    .top-sites-list .top-site-outer:hover .tile, .top-sites-list .top-site-outer:focus .tile, .top-sites-list .top-site-outer.active .tile {
+    .top-sites-list .top-site-outer:-moz-any(.active, :hover, :focus) .tile {
-    .top-sites-list .top-site-outer:hover .edit-menu, .top-sites-list .top-site-outer:focus .edit-menu, .top-sites-list .top-site-outer.active .edit-menu {
+    .top-sites-list .top-site-outer:-moz-any(.active, :hover, :focus) .edit-menu {
-    .top-sites-list .top-site-outer:hover .context-menu-button, .top-sites-list .top-site-outer:focus .context-menu-button, .top-sites-list .top-site-outer.active .context-menu-button {
+    .top-sites-list .top-site-outer:-moz-any(.active, :hover, :focus) .context-menu-button {
```

Reply:
Increased brevity is always a plus!